services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/commons/RoleTypeMapValue.java

```diff
@@ -0,0 +1,7 @@
+package com.sourcefuse.jarc.services.usertenantservice.commons;
+
+public interface RoleTypeMapValue {
```

Comment:
what is this for ?

Reply:
still pending

Reply:
Removed this as of now ,but will use it again when implementing TenantUserController

Description:
This interface is used  in a Static map as a value , 
and this static map is used in TenantUserController  -> TenantUserService -> BL to fetch the value from static map by passing key .
so after getting the  RoleTypeMapValue object -> we get the permission and have  used this permission in BL ,to check if the permision is granted to add or delete a User
Static map::
roleTypeMap.put(
      ADMIN,
      new RoleTypeMapValue() {
        public String permissionKey() {
          return "PlatformAdmin";
        }

        public int value() {
          return ADMIN.ordinal();
        }
      }
    );